receiver/kafkareceiver/metadata.yaml

```diff
@@ -18,27 +18,99 @@ tests:
       top:
         - github.com/twmb/franz-go/pkg/kfake.(*group).manage
 
+attributes:
+  topic:
+    description: The Kafka topic.
+    type: string
+  partition:
+    description: The Kafka topic partition.
+    type: int
+  node_id:
+    description: The Kafka node ID.
+    type: int
+  outcome:
+    description: The operation outcome.
+    type: string
+    enum: [success, failure]
+
 telemetry:
   metrics:
-    kafka_receiver_messages:
```

Comment:
This would be a breaking change. For the receiver we should either add a new "records" metric and keep the "messages" one but deprecate it; or alternatively update the exporter to use "messages" instead of "records". Sorry, I forgot we had this one.

I'd probably update the exporter metric for now, and we could deprecate/rename later. WDYT?